client/src/App.css

```diff
@@ -111,7 +111,16 @@ section.card div {
 .questionNum {
   font: 50px 'Helvetica', sans-serif;
 }
+
 .overlay {
   color: #bfbfbf;
   padding-left: 2px;
 }
+
+.def_glyph {
```

Comment:
http://stackoverflow.com/questions/7560813/why-are-dashes-preferred-for-css-selectors-html-attributes